### ISSUE TITLE
Test Loss and Training Loss with Regularisation

### ISSUE THREAD

**Issue by ODubrule** [tensorflow/playground#109]
Can anybody tell me what are the Training Loss and Test Loss given by the program when there is regularisation? Are they the mismatch at the data (as they should be), or are they the full loss function including the regularisation term?
Olivier
